ff/src/bits.rs

```diff
@@ -0,0 +1,82 @@
+/// Iterates over a slice of `u64` in *big-endian* order.
+#[derive(Debug)]
+pub struct BitIteratorBE<Slice: AsRef<[u64]>> {
+    s: Slice,
+    n: usize,
+}
+
+impl<Slice: AsRef<[u64]>> BitIteratorBE<Slice> {
+    pub fn new(s: Slice) -> Self {
+        let n = s.as_ref().len() * 64;
+        BitIteratorBE { s, n }
+    }
+
+    /// Construct an iterator that automatically skips any leading zeros.
+    /// That is, it skips all zeros before the most-significant one.
```

Comment:
```suggestion
    /// That is, it skips all zeros before the most-significant “one”.
```

This is a proposal for consideration because "one" here can also refer to a zero.

Reply:
Not sure I understand; do you mean that the iterator can be empty if there are no ones?

Reply:
Just realized that "most-significant one" cannot be "most-significant zero", but has to be a "one"